packages/compartment-mapper/src/link.js

```diff
@@ -328,36 +211,57 @@ const makeModuleMapHook = (
   return moduleMapHook;
 };
 
+/**
+ * @type {ImportNowHookMaker}
+ */
+const impossibleImportNowHookMaker = () => {
```

Comment:
Maybe consider `never` for `impossible`.

Reply:
I wouldn't know how to make something that's not an adjective work here.